doc/sphinx/language/core/definitions.rst

```diff
@@ -40,12 +40,22 @@ Type cast
 
 .. prodn::
    term_cast ::= @term10 : @type
+   | @term10 :> @type
    | @term10 <: @type
    | @term10 <<: @type
```

Comment:
We try to present things in the order of importance to users while also grouping similar items, to the extent possible.  Should this be #2 in the list?  If not, please reorder in `orderedGrammar` and change the order of the descriptions below to match the new order.